packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -403,26 +396,30 @@ function createProjectionAndParams({
             // Sadly need to select the whole point object due to the risk of height/z
             // being selected on a 2D point, to which the database will throw an error
             if (point) {
-                fields.push(isArray ? "point:p" : `point: ${varName}.${key}`);
+                fields.push(isArray ? "point:p" : `point: ${varName}.${field.name}`);
             }
 
             if (crs) {
-                fields.push(isArray ? "crs: p.crs" : `crs: ${varName}.${key}.crs`);
+                fields.push(isArray ? "crs: p.crs" : `crs: ${varName}.${field.name}.crs`);
             }
 
             res.projection.push(
                 isArray
-                    ? `${key}: [p in ${varName}.${key} | { ${fields.join(", ")} }]`
+                    ? `${key}: [p in ${varName}.${field.name} | { ${fields.join(", ")} }]`
                     : `${key}: { ${fields.join(", ")} }`
             );
         } else if (dateTimeField) {
             res.projection.push(
                 dateTimeField.typeMeta.array
-                    ? `${key}: [ dt in ${varName}.${key} | apoc.date.convertFormat(toString(dt), "iso_zoned_date_time", "iso_offset_date_time") ]`
-                    : `${key}: apoc.date.convertFormat(toString(${varName}.${key}), "iso_zoned_date_time", "iso_offset_date_time")`
+                    ? `${key}: [ dt in ${varName}.${field.name} | apoc.date.convertFormat(toString(dt), "iso_zoned_date_time", "iso_offset_date_time") ]`
```

Comment:
I'm assuming `key` is equivalent to `field.alias`, hence why they're interchangeable?

Reply:
That is correct. The key is whatever the field that is being requested is called. So if it has an alias then the key is equivalent to the alias. The projection produced a valid cypher request before due to these [lines](https://github.com/neo4j/graphql/blob/master/packages/graphql/src/translate/create-projection-and-params.ts#L143-L148) where it checks to see if there is an alias on the field and reassigns the key to be the field name.